packages/mdc-bottom-line/README.md

```diff
@@ -9,7 +9,7 @@ path: /catalog/input-controls/text-field/bottom-line/
 
 # Text Field Bottom Line
```

Comment:
Remove "Text Field" from this

And you'll want to change the docs comment above to something like

```
 <!--docs:
 title: "Bottom Line"
 layout: detail
 section: components
 excerpt: "The bottom line indicates where to enter text, displayed below the label"
 path: /catalog/input-controls/bottom-line/
 -->
```

(I think you can just drop the iconId, since we don't have an icon for this)